.gitignore

```diff
@@ -18,3 +18,4 @@ app/lib/core/Parsers/htmlpurifier/standalone/HTMLPurifier/DefinitionCache/Serial
 app/lib/core/Parsers/dompdf/lib/fonts/*fm.php
 app/log/log*.txt
 themes/default/views/bundles/inspector_info.php
+/install/profiles/sql
```

Comment:
If you view the whole file it's inconsistent already: https://github.com/leftclickben/providence/blob/AddProfileSQLDirectoryToGitignore/.gitignore